packages/mds-transactions-service/service/provider.ts

```diff
@@ -18,22 +22,25 @@ export const TransactionServiceProvider: ServiceProvider<TransactionService> & P
   },
   createTransactions: async transactions => {
     try {
-      return ServiceResult(await TransactionRepository.createTransactions(transactions.map(validateTransactionDomainModel)))
+      return ServiceResult(
+        await TransactionRepository.createTransactions(transactions.map(validateTransactionDomainModel))
+      )
```

Comment:
Personal preference is to assign the result of the `await` to a local const rather than use it directly on the function parameter because it allows putting a breakpoint on the `return` when debugging to see what the repository returned.